### DISCUSSION TITLE
⁉️Problem with Windows 11 22H2 v22621.1344!!

### DISCUSSION THREAD

**Discussion by jmalarcon** [jmalarcon/Win11ClockToggler#21] [Announcements]
Recently a new Windows 11 version has been released: **Windows 11 22H2 v22621.1344!!**.

If you run the app in this version you'll get this app when trying to hide the Date/Time:

![Error message, can't find the Date/Time controls](https://user-images.githubusercontent.com/1775322/224496078-8d8b73a9-6512-4779-b225-da689935cdab.png)

The problem is that they've changed the way the toolbar UI is built, so the current program it's not able to find the Date/time area, because it's located at a different place.

I've been investigating the new structure and there are **two problems**:

**1. The window/control that contains the Date/Time is located at a different level from where it used to be**. This is easily solved and I've managed to locate it in the new location (for 22H2) and keep the old code in place too, for people that has not updated their system (backwards compatibility). So it's not really a problem.
2. **It hides everything!!**: This is way worse. When you hide this control, what happens now is that **the full toolbar is hidden**, not just the Date/Time. I don't know why 🤔 So I don't know if it's possible to restore the functionality because when I hide the Date/Time, everything gets hidden for no reason... 😤

Right now, the problem is that, in the way they have designed the new toolbar in 2022H2, if I hide the date/clock, the whole toolbar is hidden. So I don't have any solutions right now except maybe trying to put something like a white rectangle above the clock or something like that. But this goes against the philosophy of this app, which is really hiding the clock.

I'm not able to find a solution for this, sorry. **I'm afraid that this is the end of the tool for now in Windows 11**, unfortunately 😢 However it **still works great with Windows 10** and is quite useful to hide it quickly before recording your screen for example.

**If someone is able to fix the problem**, please drop me a line.

If Microsoft releases a new update that gets everything back to normal, I'll update the tool to work again. 

I'll keep you posted.

There are other options around that substitute the clock for a different one (in reality, what they do is overlay something above the real one). For example [ElevenClock](https://github.com/marticliment/ElevenClock). I've not used it myself, but they say it works for changing the look and feel, and I guess you can try to hide the date/clock with it.

**Comment by itstakenallihave** [comment]
I've been using your tool and recently updated to Windows 11 Version 22H2 (OS Build 22621.1413)... I understand the tool no longer works but is there a way to revert my notification bar back to the default way (so I can see the time)? It's stuck on date only.

**Reply by jmalarcon** [reply]
Hi: that has nothing to do with my app. I guess you've modified the date format with some kind of non visible space to prevent it for showing. Just change the format to the default one. LMGTFY: https://www.askvg.com/tip-customize-date-and-time-in-windows-11-taskbar-clock/

**Reply by itstakenallihave** [reply]
> Hi: that has nothing to do with my app. I guess you've modified the date format with some kind of non visible space to prevent it for showing. Just change the format to the default one. LMGTFY: https://www.askvg.com/tip-customize-date-and-time-in-windows-11-taskbar-clock/

Thank you so much! You're right, it was completely my fault. I was so desperate to get rid of the taskbar clock with Windows 11 I forgot I tried other solutions.

Thanks for helping me fix it and I hope you're eventually able to get your program working with the updated version of Windows 11.
